apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -108,17 +106,15 @@ function getWpcomNuxPages( isGutenboarding ) {
 			alignBottom: true,
 		},
 		{
-			heading: __( 'Private until you’re ready', 'full-site-editing' ),
+			heading: __( 'Hidden until you’re ready', 'full-site-editing' ),
 			description: __(
-				'Your site will remain private as you make changes until you’re ready to launch and share with the world.',
+				'Your site will remain hidden until launched. Click “Launch” in the toolbar to share it with the world.',
 				'full-site-editing'
 			),
 			imgSrc: privateImage,
-			// @TODO: hide for sites that are already public
-			shouldHide: ! isGutenboarding,
```

Comment:
Technically thie `@TODO` hasn't been addressed, but I don't know who's responsibility it is or even when this modal would appear for a public site so 🤷

Reply:
My inclination is to let the reference to "hidden" slide for now. The main reason we're showing this one for all users I think stems from  p9Jlb4-1SN-p2 

> The intention the button will now be displayed to users regardless of their path into WordPress and will be combined with the Focused launch flow work currently in development by Team Luna.

Reply:
I'd leave the `@T0D0` because from what I understand it is referring to the launched/un-launched status of a website and not to the onboarding (site creation) flow. So here is an example where we probably don't want the slide to be displayed:
1. User created a site and landed in _My Home_ or _Plans_ (after abandoning cart).
2. User launched the site via Site setup checklist in My Home without opening the editor.
3. User navigates to the editor and the _Launch_ button in the toolbar is gone since the site is already launched.